benchmark/opperf/utils/op_registry_utils.py

```diff
@@ -437,6 +458,26 @@ def get_all_loss_operators():
     return loss_mx_operators
 
 
+def get_remaining_miscellaneous_operators():
+    """Gets remaining Miscellaneous operators registered with MXNet not covered by individual tests.
+
+    Returns
+    -------
+    {"operator_name": {"has_backward", "nd_op_handle", "params"}}
+    """
+    misc_ops = ['squeeze', 'all_finite', 'clip', 'multi_lars', 'SequenceReverse', 'SequenceLast', 'SequenceMask', 'cast_storage', 'cumsum', 'fill_element_0index']
+
+    # Get all mxnet operators
+    mx_operators = _get_all_mxnet_operators()
+
+    # Filter for Miscellaneous operators
+    misc_mx_operators = {}
+    for op_name, _ in mx_operators.items():
+        if op_name in misc_ops:
```

Comment:
This code is duplicated it could be factored out with above. also testing for element in a list is linear, better use a set.

Reply:
Good catch!

Reply:
Great insights @larroy, I’ll make those changes.